ramls/login.raml

```diff
@@ -158,13 +161,19 @@ traits:
     post:
       description: Get a new refresh and access token
       headers:
-        # The cookie will contain crumbs for the refresh token and the access token.
         Cookie:
+          description: refresh token cookie (required) and access token cookie (optional)
+          example: |
+                folioAccessToken=abc.def.ghi; Max-Age=600; Expires=Tue, 23 Jan 2024 15:01:13 GMT; Path=/; Secure; HTTPOnly; SameSite=Strict
+                folioRefreshToken=rst.uvw.xyz; Max-Age=604800; Expires=Tue, 30 Jan 2024 15:01:13 GMT; Path=/authn; Secure; HTTPOnly; SameSite=Strict
```

Comment:
The `Cookie` header does not include attributes like `Max-Age`, `Expires`, ... .
See [Cookie](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cookie) vs [Set-Cookie](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie).